shared/js/background/load.es6.js

```diff
@@ -14,6 +14,30 @@ function url (url) {
     return loadExtensionFile({ url: url, source: 'external' })
 }
 
+async function fetchData (url, fallbackUrl, headers) {
+    let response = null
+    response = await fetch(url, { method: 'GET', headers })
+    if (fallbackUrl && response.status !== 200 && response.status !== 304) {
+        response = await fetch(fallbackUrl, { method: 'GET', headers })
+    }
```

Comment:
We just have the fallbackUrl because we might fail to construct a valid URL based on UA, right? Can't we just fix URL construction and remove this complexity?